packages/agentssdk/package.json

```diff
@@ -0,0 +1,58 @@
+{
+  "name": "@microsoft/teams.agentssdk",
```

Comment:
can we change this to `@microsoft/teams.agents-sdk`?

Reply:
another typo